### DISCUSSION TITLE
Autolisp

### DISCUSSION THREAD

**Discussion by Fraiddd** [community/community#51103] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,
I regret terribly that the Autolisp language is not taken into account. (common lisp instead). 
Why? 
Autolisp is used to develop CAD applications within Autocad, the most widely used DAO/CAD software in the world. (Building, industry, roads, etc.). Hoping that one day you want to insert it.
Cordially

**Comment by look** [comment, marked as answer]
Thanks for the suggestion @Fraiddd. We use [linguist](https://github.com/github/linguist) to detect programming languages. Please file an "New language" Issue in that repository with details about Autolisp so we can improve the language detection.
